letsencrypt-auto-source/pieces/bootstrappers/deb_common.sh

```diff
@@ -18,11 +20,18 @@ BootstrapDebCommon() {
   # - Debian 6.0.10 "squeeze" (x64)
 
   $SUDO apt-get update || echo apt-get update hit problems but continuing anyway...
+  
+  # checking if certbot/letsencrypt is already packaged with OS
+  if [ "$QUIET" != 1 ]; then
+      if dpkg --get-selections | grep -v letsencrypt.sh | grep -c letsencrypt -gt 0; then
```

Comment:
I'm not a bash master, but that if condition doesn't work properly. Apparently the -gt 0 bit doesn't work as intended.